packages/cloud_functions/android/src/main/java/io/flutter/plugins/firebase/cloudfunctions/cloudfunctions/CloudFunctionsPlugin.java

```diff
@@ -32,9 +33,17 @@ public void onMethodCall(MethodCall call, final Result result) {
     switch (call.method) {
       case "CloudFunctions#call":
         String functionName = call.argument("functionName");
-        HttpsCallableReference httpsCallableReference =
-            FirebaseFunctions.getInstance().getHttpsCallable(functionName);
         Map<String, Object> parameters = call.argument("parameters");
+        String appName = call.argument("app");
+        FirebaseApp app = FirebaseApp.getInstance(appName);
```

Comment:
if `appName` is `null` do we want `FirebaseApp.getInstance()`, thats for default `FirebaseApp` as opposed to `FirebaseApp.getInstance(null)` which might throw? (it's not very clear per docs https://firebase.google.com/docs/reference/android/com/google/firebase/FirebaseApp.html#getInstance(java.lang.String))

Reply:
I think appName cannot be null. There is an `assert(name != null)` in firebase_core. The default app is named `__FIRAPP_DEFAULT` on iOS and `[DEFAULT]` on Android.